corehq/apps/translations/app_translations/utils.py

```diff
@@ -8,6 +8,7 @@
 from corehq.apps.app_manager.exceptions import ModuleNotFoundException, FormNotFoundException
 from corehq.apps.translations.const import (
     MODULES_AND_FORMS_SHEET_NAME,
+    SINGLE_SHEET_INFO_HEADERS,
```

Comment:
nitpick: `SINGLE_SHEET_HEADERS` is shorter and conveys the same information

Reply:
i used that name because these are not all the headers and only a set of all.

Reply:
How about `SINGLE_SHEET_STATIC_HEADERS` since they're the columns that don't change?